config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,17 @@
+class Rack::Attack
+  throttle('registrations', limit: 10, period: 1.day) do |req|
+    if req.path == '/' && req.post?
+      if req.env['HTTP_X_FORWARDED_FOR']
+        req.env['HTTP_X_FORWARDED_FOR'].ip.split(/\s*,\s*/)[0]
```

Comment:
The `.ip` in the middle there looks like a mistake.

Reply:
oops yep thanks